neuralmonkey/learning_utils.py

```diff
@@ -445,7 +446,9 @@ def _check_savable_dict(data):
                 else:
                     with open(path, "w", encoding="utf-8") as f_out:
                         f_out.writelines(
-                            [" ".join(sent) + "\n" for sent in data])
+                            [" ".join(sent) + "\n"
+                             if isinstance(sent, collections.Iterable)
+                             else str(sent) + "\n" for sent in data])
```

Comment:
je to správně uzávorkovaný?

Reply:
Jo, je to diff, co jsem posílal Desmondovi, aby si mohl vypsat do souboru a fungovalo to.